src/mscorlib/src/System/Globalization/NumberFormatInfo.cs

```diff
@@ -61,11 +61,11 @@ sealed public partial class NumberFormatInfo : ICloneable, IFormatProvider
         internal String numberGroupSeparator = ",";
         internal String currencyGroupSeparator = ",";
         internal String currencyDecimalSeparator = ".";
+#if FEATURE_COREFX_GLOBALIZATION
+        internal String currencySymbol = "$"; // TODO: CoreFX #846 Restore to the original value "\x00a4";  // U+00a4 is the symbol for International Monetary Fund.
```

Comment:
This looks like a hack that we forgot to cleanup. Shouldn't we rather get rid of it than keep replicating it?